docs/explore/netlify/getting-started-with-netlify/index-getting-started-with-netlify.mdx

```diff
@@ -0,0 +1,305 @@
+---
+id: index-getting-started-with-netlify
+title: Getting Started with Netlify
+sidebar_label: Getting Started with Netlify
+slug: /explore/netlify/getting-started-with-netlify
+---
+
+import RedisCard from '@site/src/theme/RedisCard';
+
+Netlify is a popular static site hosting serverless platform. It is a popular way to build, deploy, and scale modern web applications in a much more scalable and secure way.  
+
+
+Netlify helps developers to launch websites and campaigns in minutes with no-ops. The hosting platform is built on top of JAMstack and unifies JavaScript and APIs,strong enough to create an infrastructure that’s well suited for both developers and content editors.
+
+![MyImage](netlify.png)
+
+### Features of Netlify
+
+- It delivers out-of-the-box continuous integration.
+- The platform allows continuous deployment through its support for Git repository deployment.
+- It allows developers to focus on building and deploying apps by asbtracting all the maintenance work away from the developers.
+- The platform provides free SSL, CDN, and continuous integration.
+- It has built-in DNS management & SSL certificates.
+
+### How does Netlify work?
+
+![MyImages](netlifyimage.png)
+
+1. The developer writes a code and store it in a version control repository(say, GitHub).
+2. When a new change is merged into the master branch of GitHub repo, Github triggers a webhook that Netlify is listening for. 
+3. Netlify pulls the latest version of the app from GitHub and then start building the static version of the site
+4. Netlify then uses Plugins and internal code to make adjustments to your site, pre-render all of your pages in static HTML and improves it further
```

Comment:
Remove this step, it is unnecessary and Netlify doesn't do anything specific here.